staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -70,8 +70,8 @@ type AutoscaleOptions struct {
 
 	createAnnotation bool
 	args             []string
-	enforceNamespace bool
-	namespace        string
+	EnforceNamespace bool
```

Comment:
Is there any reasons why this is changed to capital letters?

Reply:
Sorry missed this comment. Yeah need not be capital letters, let me update the PR.